tracing/tracing.go

```diff
@@ -33,9 +39,32 @@ func defaultConfig() *config {
 		serviceName: envOrDefault(signalfxServiceName),
 		accessToken: envOrDefault(signalfxAccessToken),
 		url:         envOrDefault(signalfxEndpointURL),
+		globalTags:  envGlobalTags(),
+
 	}
 }
 
+// envGlobalTags extract global tags from the environment variable and parses the value in the expected format
+// key1:value1,
+func envGlobalTags() []tracer.StartOption {
+	var globalTags []tracer.StartOption
+	if val := os.Getenv(signalfxSpanTags); val != "" {
```

Comment:
nit: in golang is more idiomatic to check for `val == ""` empty and bail instead of nesting the for inside the if.